### PR TITLE
Comment Additions and Sanity Checks

### PR DESCRIPTION
**Pull request by Jahanobik234**

### CODE REVIEW

File: modules/dataset.py [hunk 1]
```diff
@@ -12,41 +13,49 @@
...
         with open('text_features_' + self.split + '.sav') as f:
             text = joblib.load(f)
 
-        self.data = dict()
+        #Open File According to Split
+        with open('./data/' + self.split + '_split.csv', 'r') as file:
+            csv_reader = csv.reader(file)
+            next(csv_reader)
 
-        with open('../data/train_split.csv', 'r') as csv_file:
```
Comment (Jahanobik234) [inline modules/dataset.py:27]: @sreekanth-sreekumar - it seemed that this file was never used after opening - any reason for it?

File: modules/dataset.py [hunk 2]
```diff
@@ -12,41 +13,49 @@
...
-                for row in csv_reader:
-                    id = row[0]
-                    
-                    # Getting audio features and length
+            #For Each Row in the Split File
+            for row in csv_reader:
+                #Get Row ID
+                id = row[0]
+
+                if id is not None:
```
Comment (Jahanobik234) [inline modules/dataset.py:40]: Just a quick check to make sure we get an ID

File: modules/dataset.py (hunk 1 above)
Comment (sreekanth-sreekumar) [reply modules/dataset.py:27]: Yeah, it needs to be used once we figure out masking. It's a placeholder for now

File: modules/dataset.py [hunk 3]
```diff
@@ -59,43 +68,45 @@ def __init__(self, split):
...
                         pad_rep = torch.zeros(max_audio_len - sample['audio_length'], sample[key].shape[1])
                         padded = torch.cat((sample[key]))
                     elif key == 'video':
-                        pad_rep = torch.zeros(max_audio_len - sample['video_length'], sample[key].shape[1])
+                        pad_rep = torch.zeros(max_video_len - sample['video_length'], sample[key].shape[1])
                         padded = torch.cat((sample[key]))
-                    
                     elif key == 'text':
-                        pad_rep = torch.zeros(max_audio_len - sample['text_length'], sample[key].shape[1])
+                        pad_rep = torch.zeros(max_text_len - sample['text_length'], sample[key].shape[1])
```
Comment (sreekanth-sreekumar) [inline modules/dataset.py:103]: Gosh ... Thanks for catching this

**Review by sreekanth-sreekumar — APPROVED** [review]
